go/obscuronode/host/host.go

```diff
@@ -73,24 +77,31 @@ type Node struct {
 
 	// Node nodeDB - stores the node public available data
 	nodeDB *DB
+
+	// A node is marked as ready after it has bootstrapped existing blocks and has the enclave secret
+	readyForWork *int32
+
+	// Handles tx conversion from eth to L1Data
+	txHandler rollupcontractlib.TxHandler
 }
 
 func NewObscuroAggregator(
 	id common.Address,
 	cfg AggregatorCfg,
-	l1 obscurocommon.L1Node,
+	l1 l1client.EthereumClient,
```

Comment:
maybe let's rename this "l1" param as well

Reply:
done 👍

Reply:
name is not consistent. Should be "ethClient"

Reply:
I was under the impression the name for the interface was EthereumClient ?
Changing it to EthClient 👍